polkadot/xcm/xcm-executor/src/lib.rs

```diff
@@ -910,6 +916,9 @@ impl<Config: config::Config> XcmExecutor<Config> {
 						&reserve,
 						Some(&mut self.holding),
 					);
+					// TODO: modify the message by adding setAssetClaimer. 
+					// replicate for teleport and regular withdraw functions.
+					// add e2e tests. 
```

Comment:
We need to decide if we want to _implicitly_ propagate the asset claimer location to forwarded XCMs, or we let the XCM builders _explicitly_ configure it for each XCM run at each hop.

Aka, do we need to add it here, or expect the caller to have added it in `xcm` param (if they wanted a custom claimer on dest as well).

I am leaning towards the latter.